guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<blockquote>There is a downside of having a 8x upsmapling in the end is that there is no learnable layers hence the results that we get are blobs as results and fails to capture high resolution detailed information.</blockquote><p> </p><p>The downside of having an "8x up-sampling decoder" is that there is no learnable layer hence the results we get are blobs that it fails to capture high-resolution information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done</p><p> </p>